tracer/src/Datadog.Trace/Telemetry/Collectors/ConfigurationTelemetry.Collector.cs

```diff
@@ -8,14 +8,28 @@
 using System.Collections.Concurrent;
 using System.Collections.Generic;
 using System.Diagnostics;
+using System.Globalization;
 using System.Threading;
+using Datadog.Trace.Logging;
 using Datadog.Trace.Telemetry;
 
 namespace Datadog.Trace.Configuration.Telemetry
 {
     // This is the "collector" implementation
     internal partial class ConfigurationTelemetry
     {
+        private static readonly IReadOnlyDictionary<string, string> ConfigurationKeysMapping = new Dictionary<string, string>
```

Comment:
But this mapping is already done in the intake? I thought we were pushing for them to use the transformed values?